coldfront/core/allocation/templates/allocation/allocation_table_view.html

```diff
@@ -0,0 +1,158 @@
+{% extends "common/base.html" %}
+{% load common_tags %}
+{% load crispy_forms_tags %}
+{% load static %}
+
+
+{% block title %}
+Allocation View
+{% endblock %}
+
+
+{% block content %}
+<h2>Table of All Allocations</h2>
```

Comment:
I put this title to make it really obvious which template was being instantiated when debugging a URL problem.

We should probably rename this to something simpler like "Allocations"